tests/support/global-setup.ts

```
@@ -241,6 +241,41 @@ spec:
+      OLS_NAMESPACE,
+      `OCP_VERSION=${process.env.OCP_VERSION}`,
+    ]);
+    console.log('Waiting for rollout to complete...');
+    oc([
+      'rollout',
+      'status',
+      `deployment/${deploymentName}`,
+      '-n',
+      OLS_NAMESPACE,
+      '--timeout=5m',
+    ]);
```

ESLint

[error] Replace `⏎······'rollout',⏎······'status',⏎······`deployment/${deploymentName}`,⏎······'-n',⏎······OLS_NAMESPACE,⏎······'--timeout=5m',⏎····` with `'rollout',·'status',·`deployment/${deploymentName}`,·'-n',·OLS_NAMESPACE,·'--timeout=5m'`

(prettier/prettier)